landing_page/app/models/ability.rb

```diff
@@ -16,12 +15,12 @@ def initialize(user)
     #     can :read, :all
     #   end
     #
-    # The first argument to `can` is the action you are giving the user 
+    # The first argument to `can` is the action you are giving the user
```

Comment:
What do you think of having to fix commented lines as a `rubucop` offense?

Reply:
It probably should ignore comments. I'll look into it 👍